.github/workflows/ci-nightly-performance-testing.yaml

```diff
@@ -0,0 +1,34 @@
+name: "NIGHTLY:EVM:PERFORMANCE:TESTING"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *' # Runs every day at midnight
```

Comment:
Nitpick: Perhaps add the time zone for clarity, as this runs every day at midnight UTC, which is 5 PM Pacific.